### ISSUE TITLE
selectOnTab is potentially confusing for screen reader users

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#3967]
### Package name(s)

Core (@salt-ds/core)

### Package version(s)

latest

### Description

Our a11y SME made a comment that selectOnTab is potentially confusing behaviour because Tab is a key that typically navigates **only**, therefore it should be disabled by default and a warning should be added to the page

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [ ] I am an employee of JPMorgan Chase & Co.

**Comment by origami-z** [comment]
what is missing? can it be made accessible? APG example is doing this?

**Comment by joshwooding** [comment]
@origami-z I'll let Jake comment but using a key that is used for navigation is the issue

**Comment by origami-z** [comment]
Ok. Will need more details on these, otherwise someone may pick up the issue to work.

https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/#accessibilityfeatures

**Comment by joshwooding** [comment]
> Ok. Will need more details on these, otherwise someone may pick up the issue to work.
> 
> [w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only#accessibilityfeatures](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/#accessibilityfeatures)

It's awaiting triage, hopefully they won't.

**Comment by joshwooding** [comment]
Relevant discussion: https://github.com/w3c/aria-practices/issues/2703

**Comment by joshwooding** [comment]
Closing at not an issue, our official stance is that we follow functionality/guidelines provided by the APG.
